lib/nimbase.h

```diff
@@ -122,6 +122,8 @@ __clang__
 */
 #if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201112 && !defined __STDC_NO_THREADS__
 #  define NIM_THREADVAR _Thread_local
+#elif defined __GENODE__
+#  define NIM_THREADVAR
```

Comment:
It can be dropped, proper TLS emulation elsewhere prevents it from being emitted.